kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/wlst/additional-build-commands-template

```diff
@@ -0,0 +1,46 @@
+#Copyright (c) 2021, Oracle and/or its affiliates.
+#Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+#
+#Adopted from https://github.com/oracle/docker-images/blob/main/OracleWebLogic/samples/12213-domain-home-in-image/Dockerfile
+# ==============================================
+[final-build-commands]
+USER oracle
+# Add files from addtionalBuildFiles required to build this image
+COPY --chown=oracle:root files/* /u01/oracle/
+
+# WLS Configuration
+# ---------------------------
+ENV ORACLE_HOME=/u01/oracle \
+    PROPERTIES_FILE_DIR="/u01/oracle/properties" \
+    DOMAIN_HOME="/u01/oracle/user_projects/domains/%DOMAIN_NAME%" \
+    DEBUG_PORT="8453" \
+    PATH=$PATH:/u01/oracle/oracle_common/common/bin:/u01/oracle/wlserver/common/bin:${DOMAIN_HOME}:${DOMAIN_HOME}/bin:/u01/oracle
+
+
+#Create directory where domain will be written to
+USER root
+RUN chmod +xw /u01/oracle/*.sh && \
+    chmod +xw /u01/oracle/*.py && \
+    mkdir -p ${PROPERTIES_FILE_DIR} && \
+    mkdir -p $DOMAIN_HOME && \
+    chown -R oracle:root $DOMAIN_HOME/.. && \
+    chown -R oracle:root ${PROPERTIES_FILE_DIR} && \
+    mv /u01/oracle/*.properties ${PROPERTIES_FILE_DIR} 
+
+
+# Configuration of WLS Domain
+USER oracle
+RUN /u01/oracle/createWLSDomain.sh && \
+    chmod -R g+w $DOMAIN_HOME && \
+    echo ". $DOMAIN_HOME/bin/setDomainEnv.sh" >> /u01/oracle/.bashrc && \
+    rm ${PROPERTIES_FILE_DIR}/*.properties
+
```

Comment:
The following lines don't look like they're needed. The sample is (correctly) using 'off-line' WLST which doesn't need the admin server...